### PR TITLE
Sidebar

### PR DESCRIPTION
**Pull request by marySalvi**
- Fixes the cesium viewer by removing all features using sprintf function
- Adds sidebar with Lat/Lng/Height functionality
- Access token config has been updated to use all base layer picker features

### CODE REVIEW

File: src/components/organisms/CesiumViewer.vue [hunk 1]
```diff
@@ -45,8 +67,6 @@ export default defineComponent({
   <vc-viewer
     class="viewer"
     base-layer-picker
-    :animation="animation"
-    :timeline="timeline"
```
[the remark this answers is not in the document]
Comment (marySalvi) [reply src/components/organisms/CesiumViewer.vue:49]: Yes, these are the features that use the Cesium.sprintf function which is broken in vue-cesium2. If we want those features moving forward we should be able to implement them again after I remove vue-cesium.

File: src/components/molecules/ToolBar.vue [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+          v-for="item in items"
+          :key="item"
+          max-width="150px"
+        >
+          {{ item }}
+        </v-tab>
+      </v-tabs>
+    </v-toolbar>
+    <v-toolbar
+      v-else-if="text"
```
[the remark this answers is not in the document]
Comment (marySalvi) [reply src/components/molecules/ToolBar.vue:47]: Sounds good to me. I went back and forth on this myself.

[the remark this answers is not in the document]
Comment (marySalvi) [reply src/components/molecules/ToolBar.vue]: This should be set now.

**Comment by mvandenburgh** [comment]
@marySalvi are the latitude/longitude/height Cesium controls supposed to work at this time? I get CORS errors thrown in my console when I try to use them. 

If not I think that's okay to handle in a future PR, just want to make sure I'm not encountering a bug that you haven't seen.

**Comment by mvandenburgh** [comment]
> @marySalvi are the latitude/longitude/height Cesium controls supposed to work at this time? I get CORS errors thrown in my console when I try to use them.
> 
> If not I think that's okay to handle in a future PR, just want to make sure I'm not encountering a bug that you haven't seen.

For the record, we talked offline and this was caused by a misconfiguration in my local environment.

**Pull request merged**